### ISSUE TITLE
Try to improve the search table editing experience

### ISSUE THREAD

**Issue by Ostrzyciel** [rdzanyM/science_points#74]
Editing the table is a horror, there's no cursor, backspace doesn't work, what I write disappears randomly. I do realize that this is caused by Dash, but can anything be done about this? Are there issues about it on Dash's tracker that suggest some workarounds? Or was this already fixed?

**Comment by rdzanyM** [comment]
I didn't find anything useful in documentation, but I found this issue https://github.com/plotly/dash-table/issues/846

**Comment by rdzanyM** [comment]
I haven't found anything useful in the documentation but I found [this issue](https://github.com/plotly/dash-table/issues/846)

**Comment by rdzanyM** [comment]
And for some reason, GitHub decided that it will link it under the mentioned issue...  
And I could not undo it by removing the comment.

**Comment by rdzanyM** [comment]
Ok, I checked it and indeed the cursor somewhat works in 4.9.0 (you have to double click on the cell and then click in a position where you want the cursor to appear). With cursor, using backspace/delete removes a single symbol and the cursor can be moved with arrows. However, if we want to use this version we have to go back to dash==1.14.0, and the syntax there is a bit different so some changes to the whole project will be necessary. Also, some features may be unavailable (like tooltip-header in datatable).

**Comment by rdzanyM** [comment]
Oh, I also found out that the cursor works in the current version as well, unfortunately, you can't set its place with a mouse click.  
If you double click on a cell you can use the cursor with all its benefits, but you have to set its position with keyboard arrows (not very intuitive)

**Comment by rdzanyM** [comment]
Also, when editing with cursor, you have to press enter to save the changes (losing focus will discard them)

**Comment by rdzanyM** [comment]
Knowing all that, editing the table is way more comfortable, but for a new user it may be a challenge to figure out how it works.

**Comment by CrimsonRadiator** [comment]
https://community.plotly.com/t/datatable-cell-text-edit/25429
TL;DR: No idea how to fix backspace
